write.go

```diff
@@ -362,6 +362,8 @@ func encodeHeaders(enc *hpack.Encoder, h *protocol.ResponseHeader, keys []string
 	// }
 
 	encKV(enc, consts.HeaderServerLower, string(h.Server()))
+	encKV(enc, consts2.HeaderEncodingLower, string(h.ContentEncoding()))
```

Comment:
Do we need process other special headers？

Reply:
No, other special header has been handled in `writeResHeaders`